R_grambank/make_theo_scores.R

```diff
@@ -12,11 +12,11 @@ feature_scores <- data.table::fread(file.path("output", "GB_wide", "parameters_b
                                     quote = "\"", header = TRUE, 
                                     sep = "\t") %>% 
   filter(Binary_Multistate != "multi") %>% 
-  dplyr::select(Parameter_ID = ID, Name, Flexivity,`locus of marking`, `word order`, `Gender/noun class`, informativity) %>% 
```

Comment:
these changes are surprising to me, i'll have to check.

Reply:
checked, they make sense. That change must be recent, the colnames used to be different.